core/startos/src/util/docker.rs

```diff
@@ -115,7 +115,9 @@ pub async fn stop_container(
     let mut cmd = Command::new(CONTAINER_TOOL);
     cmd.arg("stop");
     if let Some(dur) = timeout {
-        cmd.arg("-t").arg(dur.as_secs().to_string());
+        cmd.arg("-t")
+            .arg(dur.as_secs().to_string())
+            .timeout(Some(dur));
```

Comment:
This won't work, timeout lifts it into a wrapper type. You'll need to reassign cmd